### PR TITLE
fix: active_campaign error handler

### PR DESCRIPTION
**Pull request by sanpj2292**
## What are the changes introduced in this PR?

Whenever there are node errors like EAI_AGAIN, ECONNABORTED etc,. the error handler is not able to handle the error properly.
Through this PR we aim to fix it.

Resolves INT-1157

## Please explain the objectives of your changes below

N/A

### Type of change

If the pull request is a **bug-fix**, **enhancement** or a **refactor**, please fill in the details on the changes made.

- Existing capabilities/behavior

If the pull request is a **new feature**,

### Any technical or performance related pointers to consider with the change?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### If the PR has changes in more than 10 files, please mention why the changes were not split into multiple PRs.

N/A

### If multiple linear tasks are associated with the PR changes, please elaborate on the reason:

N/A

<hr>

### Developer checklist

- [x] **No breaking changes are being introduced.**

- [ ] Are all related docs linked with the PR?

- [x] Are all changes manually tested?

- [ ] Does this change require any documentation changes?

- [x] Are relevant unit and component test-cases added?

### Reviewer checklist

- [x] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Review by yashasvibajpai — DISMISSED** [review]

**Review by koladilip — APPROVED** [review]

**Review by yashasvibajpai — APPROVED** [review]

File: src/v0/destinations/active_campaign/transform.js [hunk 1]
```diff
@@ -62,11 +62,11 @@ const syncContact = async (contactPayload, category, destination) => {
     feature: 'transformation',
   });
   if (res.success === false) {
-    errorHandler(res.response, 'Failed to create new contact');
+    errorHandler(res, 'Failed to create new contact');
```
Comment (utsabc) [inline src/v0/destinations/active_campaign/transform.js:65]: Doesn't this need update in `errorHandler` logic as well?

Comment (utsabc) [reply src/v0/destinations/active_campaign/transform.js:65]: If not how was it working previously?

Comment (sanpj2292) [reply src/v0/destinations/active_campaign/transform.js:65]: The `res` variable is of type `AxiosError` which contains `response`. And this `response` contains the details of the error in field `data`

Basically `res.response.data` contains the error response from `active_campaign`
Please see the screenshot for more clarification

Screenshot 2023-12-12 at 13 14 28

Comment (sanpj2292) [reply src/v0/destinations/active_campaign/transform.js:65]: For success, we are dealing normally & don't need to depend on `errorHandler` logic.

**Review by koladilip — APPROVED** [review]

File: src/v0/destinations/active_campaign/transform.js (hunk 1 above)
Comment (utsabc) [reply src/v0/destinations/active_campaign/transform.js:65]: ok

**Pull request merged**